### ISSUE TITLE
List Routes does not show routes with .domain()

### ISSUE THREAD

**Issue by joelcieslar** [Julien-R44/adonis-vscode-extension#7]
The "list routes" table is missing routes that use `.domain()`, e.g.

```
Route.get('/', ({ view }) => view.render('landing/index'))
Route.get('/caddy/allowed-domain', 'TenantsController.checkAllowedDomain')

// Tenant routes
Route.group(() => {
    Route.get('/admin/reports', 'ReportsController.index')
    Route.get('/admin/sessions', 'SessionsController.index')
    Route.get('/admin/tickets', 'TicketsController.index')
})
  .domain(`:tenantSlug.${Env.get('BASE_DOMAIN')}`) // 👈 without this line, it seems to work fine
  .middleware(['auth:web'])
```

### The result
Note that the following routes are missing:
- `/admin/reports`
- `/admin/sessions`
- `/admin/tickets`

![image](https://user-images.githubusercontent.com/18103212/175055142-d37f112b-9795-4edb-93ff-53b50b293669.png)

### Package versions
- adonis-vscode-extension - `v0.3.0`
- Node.js - `v16.15.0`
- NPM - `v8.5.5`

**Comment by Julien-R44** [comment]
Hey, thanks for reporting !

This is a feature I voluntarily skipped in the v1 release. If you feel like doing it, I'll voluntarily merge a PR
Otherwise, I'll fix that it when I have some time, I'm pretty busy right now

**Comment by Julien-R44** [comment]
Fixed in 2b8bd8e
